packages/edit-site/src/components/sidebar-navigation-root/screen-main.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalItemGroup as ItemGroup,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalNavigatorScreen as NavigatorScreen,
+	__experimentalNavigatorButton as NavigatorButton,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { layout, symbolFilled } from '@wordpress/icons';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useViewportMatch } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import SidebarNavigationTitle from '../sidebar-navigation-title';
+import { useLink } from '../routes/link';
+import SidebarNavigationItem from '../sidebar-navigation-item';
+import { useLocation } from '../routes';
+import { store as editSiteStore } from '../../store';
+import getIsListPage from '../../utils/get-is-list-page';
+
+export default function SidebarNavigationScreenMain() {
+	const { params } = useLocation();
+	const templateId = useSelect( ( select ) => {
+		return select( editSiteStore ).getEditedPostId();
+	} );
+	const isListPage = getIsListPage( params );
+	const isEditorPage = ! isListPage;
+	const templatesLink = useLink( {
+		postType: 'wp_template',
+		postId: templateId,
```

Comment:
@youknowriad do we always have an edited post?

If `undefined`, the "manage all templates" will be load, so it's fine for now. However, I understand we may want to actually get rid of that screen if the actions become embedded in the sidebar. Hence, my question. For the record, I was only able to reproduce this (loading `http://localhost:8888/wp-admin/site-editor.php?postType=wp_template`) by manually edit the URL parameters.

Reply:
Not always I think, when you're on the "template list" page, there's no template Id. We might have other similar screens in the future.

Reply:
So, if we plan to keep a "template list" page around, there's no need to do anything here (finding an alternative template id, if there is none being edited).